python/paddle/fluid/incubate/fleet/parameter_server/pslib/__init__.py

```diff
@@ -485,7 +501,7 @@ def minimize(self,
                           parameter_list,
                           no_grad_set,
                           self._strategy)
-
+        opt_info["mpi_rank"] = fleet._role_maker._get_rank()
```

Comment:
maybe trainer_rank or trainer_id is better，role_maker can have many types (mpi or paddlecloud, etc.)  and  all of them can get_rank